website/docs/r/media_asset_filter.html.markdown

```diff
@@ -0,0 +1,173 @@
+---
+subcategory: "Media"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_media_asset_filter"
+description: |-
+  Manages an Asset Filter.
+---
+
+# azurerm_media_asset_filter
+
+Manages an Asset Filter.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "media-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_storage_account" "example" {
+  name                     = "examplestoracc"
+  resource_group_name      = azurerm_resource_group.example.name
+  location                 = azurerm_resource_group.example.location
+  account_tier             = "Standard"
+  account_replication_type = "GRS"
+}
+
+resource "azurerm_media_services_account" "example" {
+  name                = "examplemediaacc"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  storage_account {
+    id         = azurerm_storage_account.example.id
+    is_primary = true
+  }
+}
+
+resource "azurerm_media_asset" "example" {
+  name                        = "Asset1"
+  resource_group_name         = azurerm_resource_group.example.name
+  media_services_account_name = azurerm_media_services_account.example.name
+  description                 = "Asset description"
+}
+
+resource "azurerm_media_asset_filter" "example" {
+  name                        = "Filter1"
+  resource_group_name         = azurerm_resource_group.example.name
+  media_services_account_name = azurerm_media_services_account.example.name
+  asset_name                  = azurerm_media_asset.example.name
+  first_quality_bitrate       = 128000
+
+  presentation_time_range {
+    start_timestamp              = 0
+    end_timestamp                = 170000000
+    presentation_window_duration = 9223372036854775000
+    live_backoff_duration        = 0
+    timescale                    = 10000000
+    force_end_timestamp          = false
+  }
+
+  track {
+    selection {
+      property  = "Type"
+      operation = "Equal"
+      value     = "Audio"
+    }
+
+    selection {
+      property  = "Language"
+      operation = "NotEqual"
+      value     = "en"
+    }
+
+    selection {
+      property  = "FourCC"
+      operation = "NotEqual"
+      value     = "EC-3"
+    }
+  }
+
+
+  track {
+    selection {
+      property  = "Type"
+      operation = "Equal"
+      value     = "Video"
+    }
+
+    selection {
+      property  = "Bitrate"
+      operation = "Equal"
+      value     = "3000000-5000000"
+    }
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `asset_name` - (Required) The Asset name. Changing this forces a new Asset Filter to be created.
```

Comment:
could we go over these docs as the these from the SDK/docs are a bit sparse
```suggestion
* `asset_name` - (Required) The Asset name to filter. Changing this forces a new Asset Filter to be created.
```

Reply:
Ok